addons/skin.estuary/xml/MyPVRRecordings.xml

```diff
@@ -49,7 +49,7 @@
 								<right>30</right>
 								<aligny>center</aligny>
 								<scroll>true</scroll>
-								<label>$INFO[ListItem.Label]$INFO[ListItem.EpisodeName, (,)]</label>
+                <label>$INFO[ListItem.Label,, ]$INFO[ListItem.EpisodeName,(,) ]$INFO[Listitem.Season,,.]$INFO[Listitem.Episode]</label>
```

Comment:
nitpick: don't postfix the label with a space, but prefix the episodename with one